commit 37b29c32c
Author: Jan Beich <jbeich@FreeBSD.org>
Date:   Fri Jan 7 01:53:52 2022 +0000

    meson: drop unused argument for i18n.merge_file()
    
    Ignored in Meson < 0.60.0, deprecated since 0.60.1 and fatal since 0.61.0.
    
    panels/applications/meson.build:10:5: ERROR: Function does not take positional arguments.
    panels/background/meson.build:10:5: ERROR: Function does not take positional arguments.
    panels/camera/meson.build:10:5: ERROR: Function does not take positional arguments.
    [...]

diff --git a/panels/applications/meson.build b/panels/applications/meson.build
index d511bf640..bd4fd6442 100644
--- a/panels/applications/meson.build
+++ b/panels/applications/meson.build
@@ -8,7 +8,6 @@ desktop_in = configure_file(
 )
 
 i18n.merge_file(
-       desktop,
          type : 'desktop',
         input : desktop_in,
        output : desktop,
diff --git a/panels/background/meson.build b/panels/background/meson.build
index 226547232..b2ee1df16 100644
--- a/panels/background/meson.build
+++ b/panels/background/meson.build
@@ -8,7 +8,6 @@ desktop_in = configure_file(
 )
 
 i18n.merge_file(
-  desktop,
   type: 'desktop',
   input: desktop_in,
   output: desktop,
diff --git a/panels/bluetooth/meson.build b/panels/bluetooth/meson.build
index c3fce7467..4bed812a9 100644
--- a/panels/bluetooth/meson.build
+++ b/panels/bluetooth/meson.build
@@ -8,7 +8,6 @@ desktop_in = configure_file(
 )
 
 i18n.merge_file(
-  desktop,
   type: 'desktop',
   input: desktop_in,
   output: desktop,
diff --git a/panels/camera/meson.build b/panels/camera/meson.build
index 4f2611879..bf2e727cf 100644
--- a/panels/camera/meson.build
+++ b/panels/camera/meson.build
@@ -8,7 +8,6 @@ desktop_in = configure_file(
 )
 
 i18n.merge_file(
-  desktop,
   type: 'desktop',
   input: desktop_in,
   output: desktop,
diff --git a/panels/color/meson.build b/panels/color/meson.build
index 156df6e47..9368ae97f 100644
--- a/panels/color/meson.build
+++ b/panels/color/meson.build
@@ -8,7 +8,6 @@ desktop_in = configure_file(
 )
 
 i18n.merge_file(
-  desktop,
   type: 'desktop',
   input: desktop_in,
   output: desktop,
diff --git a/panels/datetime/meson.build b/panels/datetime/meson.build
index 65683941f..9bbdc7ce0 100644
--- a/panels/datetime/meson.build
+++ b/panels/datetime/meson.build
@@ -8,7 +8,6 @@ desktop_in = configure_file(
 )
 
 i18n.merge_file(
-  desktop,
   type: 'desktop',
   input: desktop_in,
   output: desktop,
@@ -20,7 +19,6 @@ i18n.merge_file(
 polkit = 'org.gnome.controlcenter.@0@.policy'.format(cappletname)
 
 i18n.merge_file(
-  polkit,
   input: polkit + '.in',
   output: polkit,
   po_dir: po_dir,
diff --git a/panels/default-apps/meson.build b/panels/default-apps/meson.build
index 68550062d..2a24321a0 100644
--- a/panels/default-apps/meson.build
+++ b/panels/default-apps/meson.build
@@ -8,7 +8,6 @@ desktop_in = configure_file(
 )
 
 i18n.merge_file(
-  desktop,
   type: 'desktop',
   input: desktop_in,
   output: desktop,
diff --git a/panels/diagnostics/meson.build b/panels/diagnostics/meson.build
index fac9b4cd8..2f25c2194 100644
--- a/panels/diagnostics/meson.build
+++ b/panels/diagnostics/meson.build
@@ -8,7 +8,6 @@ desktop_in = configure_file(
 )
 
 i18n.merge_file(
-  desktop,
   type: 'desktop',
   input: desktop_in,
   output: desktop,
diff --git a/panels/display/meson.build b/panels/display/meson.build
index 97db0ebdc..726a898b9 100644
--- a/panels/display/meson.build
+++ b/panels/display/meson.build
@@ -8,7 +8,6 @@ desktop_in = configure_file(
 )
 
 i18n.merge_file(
-  desktop,
   type: 'desktop',
   input: desktop_in,
   output: desktop,
diff --git a/panels/info-overview/meson.build b/panels/info-overview/meson.build
index b941f0833..1cc062eee 100644
--- a/panels/info-overview/meson.build
+++ b/panels/info-overview/meson.build
@@ -8,7 +8,6 @@ desktop_in = configure_file(
 )
 
 i18n.merge_file(
-  desktop,
   type: 'desktop',
   input: desktop_in,
   output: desktop,
diff --git a/panels/keyboard/meson.build b/panels/keyboard/meson.build
index 68c5eae3d..991fb8122 100644
--- a/panels/keyboard/meson.build
+++ b/panels/keyboard/meson.build
@@ -8,7 +8,6 @@ desktop_in = configure_file(
 )
 
 i18n.merge_file(
-  desktop,
   type: 'desktop',
   input: desktop_in,
   output: desktop,
@@ -45,7 +44,6 @@ xml_files = [
 
 foreach file: xml_files
   i18n.merge_file(
-    file,
     input: file + '.in',
     output: file,
     po_dir: po_dir,
diff --git a/panels/location/meson.build b/panels/location/meson.build
index bfc6e82ad..f4c835d92 100644
--- a/panels/location/meson.build
+++ b/panels/location/meson.build
@@ -8,7 +8,6 @@ desktop_in = configure_file(
 )
 
 i18n.merge_file(
-  desktop,
   type: 'desktop',
   input: desktop_in,
   output: desktop,
diff --git a/panels/lock/meson.build b/panels/lock/meson.build
index 0661af4fa..18dbfb2b3 100644
--- a/panels/lock/meson.build
+++ b/panels/lock/meson.build
@@ -8,7 +8,6 @@ desktop_in = configure_file(
 )
 
 i18n.merge_file(
-  desktop,
   type: 'desktop',
   input: desktop_in,
   output: desktop,
diff --git a/panels/microphone/meson.build b/panels/microphone/meson.build
index e0847a4c4..bc7f73a20 100644
--- a/panels/microphone/meson.build
+++ b/panels/microphone/meson.build
@@ -8,7 +8,6 @@ desktop_in = configure_file(
 )
 
 i18n.merge_file(
-  desktop,
   type: 'desktop',
   input: desktop_in,
   output: desktop,
diff --git a/panels/mouse/meson.build b/panels/mouse/meson.build
index 357aaae81..7b46097f4 100644
--- a/panels/mouse/meson.build
+++ b/panels/mouse/meson.build
@@ -8,7 +8,6 @@ desktop_in = configure_file(
 )
 
 i18n.merge_file(
-  desktop,
   type: 'desktop',
   input: desktop_in,
   output: desktop,
diff --git a/panels/multitasking/meson.build b/panels/multitasking/meson.build
index 772b63813..53fd0d8ea 100644
--- a/panels/multitasking/meson.build
+++ b/panels/multitasking/meson.build
@@ -8,7 +8,6 @@ desktop_in = configure_file(
 )
 
 i18n.merge_file(
-  desktop,
   type: 'desktop',
   input: desktop_in,
   output: desktop,
diff --git a/panels/network/meson.build b/panels/network/meson.build
index d7cd6e419..43a569092 100644
--- a/panels/network/meson.build
+++ b/panels/network/meson.build
@@ -25,7 +25,6 @@ foreach name: panel_names
   )
 
   i18n.merge_file(
-    desktop,
     type: 'desktop',
     input: desktop_in,
     output: desktop,
diff --git a/panels/notifications/meson.build b/panels/notifications/meson.build
index df702614a..1b04b243d 100644
--- a/panels/notifications/meson.build
+++ b/panels/notifications/meson.build
@@ -8,7 +8,6 @@ desktop_in = configure_file(
 )
 
 i18n.merge_file(
-  desktop,
   type: 'desktop',
   input: desktop_in,
   output: desktop,
diff --git a/panels/online-accounts/meson.build b/panels/online-accounts/meson.build
index 82f1d569a..687097724 100644
--- a/panels/online-accounts/meson.build
+++ b/panels/online-accounts/meson.build
@@ -8,7 +8,6 @@ desktop_in = configure_file(
 )
 
 i18n.merge_file(
-  desktop,
   type: 'desktop',
   input: desktop_in,
   output: desktop,
diff --git a/panels/power/meson.build b/panels/power/meson.build
index 0e3ea1d56..1cafe2a1b 100644
--- a/panels/power/meson.build
+++ b/panels/power/meson.build
@@ -8,7 +8,6 @@ desktop_in = configure_file(
 )
 
 i18n.merge_file(
-  desktop,
   type: 'desktop',
   input: desktop_in,
   output: desktop,
diff --git a/panels/printers/meson.build b/panels/printers/meson.build
index ccce64cb7..042c67acc 100644
--- a/panels/printers/meson.build
+++ b/panels/printers/meson.build
@@ -8,7 +8,6 @@ desktop_in = configure_file(
 )
 
 i18n.merge_file(
-  desktop,
   type: 'desktop',
   input: desktop_in,
   output: desktop,
diff --git a/panels/region/meson.build b/panels/region/meson.build
index 68b7f000e..27bc5ff33 100644
--- a/panels/region/meson.build
+++ b/panels/region/meson.build
@@ -8,7 +8,6 @@ desktop_in = configure_file(
 )
 
 i18n.merge_file(
-  desktop,
   type: 'desktop',
   input: desktop_in,
   output: desktop,
diff --git a/panels/removable-media/meson.build b/panels/removable-media/meson.build
index b51799af9..fc07c48e5 100644
--- a/panels/removable-media/meson.build
+++ b/panels/removable-media/meson.build
@@ -8,7 +8,6 @@ desktop_in = configure_file(
 )
 
 i18n.merge_file(
-  desktop,
   type: 'desktop',
   input: desktop_in,
   output: desktop,
diff --git a/panels/search/meson.build b/panels/search/meson.build
index 39b076bd8..a4dd79d96 100644
--- a/panels/search/meson.build
+++ b/panels/search/meson.build
@@ -8,7 +8,6 @@ desktop_in = configure_file(
 )
 
 i18n.merge_file(
-  desktop,
   type: 'desktop',
   input: desktop_in,
   output: desktop,
diff --git a/panels/sharing/meson.build b/panels/sharing/meson.build
index 035883950..99b15620d 100644
--- a/panels/sharing/meson.build
+++ b/panels/sharing/meson.build
@@ -8,7 +8,6 @@ desktop_in = configure_file(
 )
 
 i18n.merge_file(
-  desktop,
   type: 'desktop',
   input: desktop_in,
   output: desktop,
@@ -29,7 +28,6 @@ polkit_in = configure_file(
 )
 
 i18n.merge_file(
-  polkit,
   input: polkit_in,
   output: polkit,
   po_dir: po_dir,
diff --git a/panels/sound/meson.build b/panels/sound/meson.build
index fa0f1281f..18f75dd4f 100644
--- a/panels/sound/meson.build
+++ b/panels/sound/meson.build
@@ -8,7 +8,6 @@ desktop_in = configure_file(
 )
 
 i18n.merge_file(
-  desktop,
   type: 'desktop',
   input: desktop_in,
   output: desktop,
diff --git a/panels/thunderbolt/meson.build b/panels/thunderbolt/meson.build
index 796524a1f..79291b003 100644
--- a/panels/thunderbolt/meson.build
+++ b/panels/thunderbolt/meson.build
@@ -8,7 +8,6 @@ desktop_in = configure_file(
 )
 
 i18n.merge_file(
-  desktop,
   type: 'desktop',
   input: desktop_in,
   output: desktop,
diff --git a/panels/universal-access/meson.build b/panels/universal-access/meson.build
index 17015c08b..b7e5a4196 100644
--- a/panels/universal-access/meson.build
+++ b/panels/universal-access/meson.build
@@ -8,7 +8,6 @@ desktop_in = configure_file(
 )
 
 i18n.merge_file(
-  desktop,
   type: 'desktop',
   input: desktop_in,
   output: desktop,
diff --git a/panels/usage/meson.build b/panels/usage/meson.build
index 74f720ecd..6047ce0ec 100644
--- a/panels/usage/meson.build
+++ b/panels/usage/meson.build
@@ -8,7 +8,6 @@ desktop_in = configure_file(
 )
 
 i18n.merge_file(
-  desktop,
   type: 'desktop',
   input: desktop_in,
   output: desktop,
diff --git a/panels/user-accounts/meson.build b/panels/user-accounts/meson.build
index 895b0dfe0..19d70fb59 100644
--- a/panels/user-accounts/meson.build
+++ b/panels/user-accounts/meson.build
@@ -8,7 +8,6 @@ desktop_in = configure_file(
 )
 
 i18n.merge_file(
-  desktop,
   type: 'desktop',
   input: desktop_in,
   output: desktop,
@@ -85,7 +84,6 @@ meson.add_install_script('sh', '-c',
 polkit = 'org.gnome.controlcenter.@0@.policy'.format(cappletname)
 
 i18n.merge_file(
-  polkit,
   input: polkit + '.in',
   output: polkit,
   po_dir: po_dir,
diff --git a/panels/wacom/meson.build b/panels/wacom/meson.build
index 30bbb6b04..7b03ce9e8 100644
--- a/panels/wacom/meson.build
+++ b/panels/wacom/meson.build
@@ -30,7 +30,6 @@ desktop_in = configure_file(
 )
 
 i18n.merge_file(
-       desktop,
          type : 'desktop',
         input : desktop_in,
        output : desktop,
diff --git a/panels/wwan/meson.build b/panels/wwan/meson.build
index 7a33ac5c8..c8ecef6d5 100644
--- a/panels/wwan/meson.build
+++ b/panels/wwan/meson.build
@@ -12,7 +12,6 @@ desktop_in = configure_file(
 )
 
 i18n.merge_file(
-       desktop,
          type : 'desktop',
         input : desktop_in,
        output : desktop,
diff --git a/shell/appdata/meson.build b/shell/appdata/meson.build
index 3b36171fb..fdb827969 100644
--- a/shell/appdata/meson.build
+++ b/shell/appdata/meson.build
@@ -1,7 +1,6 @@
 appdata = 'gnome-control-center.appdata.xml'
 
 i18n.merge_file(
-  appdata,
   input: appdata + '.in',
   output: appdata,
   po_dir: po_dir,
diff --git a/shell/meson.build b/shell/meson.build
index 2a6cbf46f..56ec21249 100644
--- a/shell/meson.build
+++ b/shell/meson.build
@@ -23,7 +23,6 @@ desktop_in = configure_file(
 )
 
 i18n.merge_file(
-       desktop,
          type : 'desktop',
         input : desktop_in,
        output : desktop,
diff --git a/tests/interactive-panels/applications/meson.build b/tests/interactive-panels/applications/meson.build
index 2657f9588..82eed4d5e 100644
--- a/tests/interactive-panels/applications/meson.build
+++ b/tests/interactive-panels/applications/meson.build
@@ -12,7 +12,6 @@ desktop_files = [
 
 foreach desktop_file : desktop_files
   i18n.merge_file(
-              desktop,
                 type : 'desktop',
                input : 'gtp-@0@.desktop.in'.format(desktop_file),
               output : 'gnome-@0@-panel.desktop'.format(desktop_file),
